ui/raidboss/data/06-ew/trial/endsinger-ex.ts

```diff
@@ -106,6 +86,51 @@ const getAoEOrbSafeDir = (posX: number, posY: number, output: Output): string |
   return output.nw!();
 };
 
+const getStarPositionFromHeading = (heading: string) => {
+  switch (parseFloat(heading)) {
```

Comment:
Sorry for not saying this the first time around, but would you mind doing some floating point comparison here (and also on the zero version)? Or doing something more like the "round a heading into an intercardinal direction" approach here? I can buy that this all works now, but mostly I'd prefer to future proof this around level of precision and not doing a string comparison against a floating point number.

Reply:
Updated in latest commit, I think this is what you were suggesting?